src/app/TASKS/task-details/task-details.component.ts

```diff
@@ -56,13 +64,20 @@ export class TaskDetailsComponent implements OnInit, OnDestroy {
               title: this.task.title,
               description: this.task.description,
               type: this.task.type,
-              status: this.task.status
+              status: this.task.status,
+              assignedTo: this.task.assignedTo || 'UNASSIGNED'
             });
           }
         }
       });
   }
-
+  fetchUsers() {
+    this.userService.getAllUsers()
+    .pipe(takeUntil(this.destroy$))
```

Comment:
you need the data once so there is not point in continuing subscription after receiving data

Reply:
instead you can use rxjs take  or first operator